components/studio/templates/guide_get_account.html

```diff
@@ -61,21 +44,15 @@ <h4 id="stepslogin"><span>Step-by-step guide</span></h4>
     <li>Click <span style="font-weight: 500;color: #041c39c7;">'Continue'</span> on the
       subsequent page indicating that you are a member of ELIXIR.
     </li>
-    <li>The next page will summarise the information to be shared with SciLifeLab Serve. Click
+    <li>Double check the summarized information. Click
       <span style="font-weight: 500;color: #041c39c7;">'Consent'</span> to proceed.
     </li>
-    <li>Early access users of SciLifeLab Serve will be given the option to add the information
-      to their exiting account. Click <span style="font-weight: 500;color: #041c39c7;">'Add to
-                                    existing
-                                    account'</span> to do so. Otherwise, complete the subsequent form to get an account.
-    </li>
-    <li><span style="font-weight: 500;color: #041c39c7;">Verify your email address</span>
-      (following instructions in the verification email you received) to link your account
-      with ELIXIR-broker.
-    </li>
-    <li>You will be redirected to your SciLifeLab Serve account.</li>
-    <li>You will receive an email regarding an access consent for your LS-login that you can
-      review.
-    </li>
+    <li>You will now be redirected to your SciLifeLab Serve account. For help creating a project, see the <a href="{% url 'guide_create_projects' %}">Projects</a> guide. 
   </ol>
+
+  <h4>Troubleshooting</h4>
```

Comment:
Yes that's true. My idea was to write more stuff here that are more of a "Troubleshooting" nature, but i'm not sure what can go wrong since it worked smoothly for me. Maybe Liane can weigh in on this aswell.